slices/libssh2-1.yaml

```diff
@@ -0,0 +1,16 @@
+# Libssh2is a client-side C library implementing the SSH2 protocol.
+# It supports regular terminal, SCP and SFTP (v1-v5) sessions;
+# port forwarding, X11 forwarding; password, key-based and
+# keyboard-interactive authentication.
+
```

Comment:
nit: let's remove this blank link to have the comment on top of the `package` field.